src/app/modules/group/pages/managed-groups/managed-groups.component.html

```diff
@@ -1 +1,9 @@
-<p>managed-groups works!</p>
+<div class="group-info-container">
+  <div class="group-info-header">
+    <span class="group-title">{{ title }}</span>
+    <span class="group-subtitle">Here are the groups you manage</span>
```

Comment:
This subtitle is useless, you can remove it.

Reply:
Done